addons/website/models/website.py

```diff
@@ -208,23 +208,25 @@ def write(self, values):
             self.env['ir.qweb'].clear_caches()
 
         if 'cookies_bar' in values:
+            existing_policy_page = self.env['website.page'].search([
+                ('website_id', '=', self.id),
+                ('url', '=', '/cookie-policy'),
+            ])
             if values['cookies_bar']:
                 cookies_view = self.env.ref('website.cookie_policy', raise_if_not_found=False)
                 if cookies_view:
                     cookies_view.with_context(website_id=self.id).write({'website_id': self.id})
                     specific_cook_view = self.with_context(website_id=self.id).viewref('website.cookie_policy')
-                    self.env['website.page'].create({
-                        'is_published': True,
-                        'website_indexed': False,
-                        'url': '/cookie-policy',
-                        'website_id': self.id,
-                        'view_id': specific_cook_view.id,
-                    })
+                    if not existing_policy_page:
```

Comment:
This is fixing the issue but is still too late, it still copy the view
Reversing the top level condition, this new condition could be merged with it 
```python
if not values['cookies_bar']:
    existing_policy_page.unlink()
elif not existing_policy_page:
   # [insert current code]
```